instagram-app/src/components/PostContainer/PostPage.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import '../../App.css';
+import dummyData from '../../dummy-data'
+import PostContainer from '../PostContainer/PostContainer';
+import SearchBar from '../SearchBar/SearchBar';
+
+
+class App extends Component {
+    constructor() {
+        super();
+        this.state = {
+            posts: [],
+            filteredPosts: []
+        };
+    }
+
+
+    componentDidMount() {
+        this.setState({ posts: dummyData });
+
+    }
+    searchPostsHandler = e => {
+        const posts = this.state.posts.filter(p => {
+            if (p.username.includes(e.target.value)) {
```

Comment:
the search function works well right now but we can make it better. It doesn't check for casing which we can do by doing the following: 
(p.username.toLowerCase().includes(e.target.value.toLowerCase())